### DISCUSSION TITLE
Way to pass expanded paramaters in filteredstream?

### DISCUSSION THREAD

**Discussion by wraithsec** [twitivity/twitter-stream.py#20] [Q&A]
So I noticed in all other examples, simply subclassing the *Stream class and adding the appropriate query params as variables would make the stream work.  However, in FilteredStream this seems like it won't work properly, as I still get the default tweet object output back when connecting to the stream.  

Does this actually work and I am being dense? or is it something not yet implemented for filteredstream yet?

**Comment by wraithsec** [comment]
I figured it out, editied the FilteredStream class to contain `_has_params = True` and it works now :).

**Reply by saadmanrafat** [reply]
We work to make life easier for developers. If you have to figure it out -- we are doing something wrong. We cater to data scientists and academics So that you don't have to worry about retrieving the data. 

Thanks for pointing this point. I'm creating a new issue to enhance the documentations. Also getting data will hopefully get a lot easier as soon we release [this CLI Tool](https://twitter.com/twitivitydev/status/1467477714290495488).
